repo2docker/buildpacks/python/__init__.py

```diff
@@ -136,8 +136,17 @@ def get_assemble_scripts(self):
     def detect(self):
         """Check if current repo should be built with the Python 3 Build pack.
         """
-        return (os.path.exists(self.binder_path('requirements.txt')) and
-                super().detect())
+        requirements_txt = self.binder_path('requirements.txt')
+        runtime_txt = self.binder_path('runtime.txt')
+
+        if os.path.exists(runtime_txt):
+            with open(runtime_txt) as f:
+                runtime = f.read().strip()
+            if runtime.startswith("python"):
```

Comment:
This will also match `python-2.7`. Should we instead check for `startswith("python-3")`?